java/src/jmri/util/StringUtil.java

```diff
@@ -711,6 +713,20 @@ public static String concatTextHtmlAware(String baseText, String extraText) {
         return result;
     }
 
+    /**
+     * Removes HTML tags from a String.
+     * Replaces HTML line breaks with newline characters from a given input string.
+     *
+     * @param originalText The input string that may contain HTML tags.
+     * @return A cleaned string with HTML tags removed.
+     */
+    public static String stripHtmlTags( final String originalText) {
+        String replaced = originalText.replace("<br>", System.lineSeparator());
```

Comment:
We also see occasional uses of `<br/>` (note slash), would it make sense to handle that too?

Reply:
Absolutely Bob, also added the `<br />` form